.github/workflows/privacy.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           path: android-tests-report.zip
 
       - name: Create Asana task when workflow failed
-        if: ${{ failure() }}
+        if: ${{ failure() && github.event_name != 'workflow_dispatch' }}
```

Comment:
Checking my understanding here, does this just mean we _don't_ create an Asana task if the job was manually created?